.github/workflows/pr-build.yml

```diff
@@ -1,21 +1,19 @@
 # This workflow will build a Java project with Gradle
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
 
-name: Java CI with Gradle
-
+name: PR Build
 on:
-  push:
-    branches: [ terraform ]
   pull_request:
-    branches: [ terraform ]
+    branches: 
+      - terraform
+      - release/v*
+      - dev/*
```

Comment:
Are we going to maintain dev branch.

Reply:
There is no maintenance required. I added this because I plan on doing some work on a dev branch and wanted to ensure that my PRs get all unit tests ran against them. Other developers can use the `dev/*` branch naming schemes in the future if they would like.